### ISSUE TITLE
Should we expose `_action_cable_internal` as a constant?

### ISSUE THREAD

**Issue by ioquatix** [anycable/actioncable-next#2]
Even thought it's kind of an internal detail, it's actually part of the public interface of Redis integration.

https://github.com/anycable/actioncable-next/blob/f9c1c2271107e25061371bfd591108282cf7109f/lib/action_cable/subscription_adapter/redis.rb#L95

e.g.

```
CHANNEL = "_action_cable_internal"
```

**Comment by ioquatix** [comment]
To clarify a bit more, I think that my own implementation of the `AsyncCableRedis` adapter should use the same channel as it's going to be integrating with the same message bus.

**Comment by palkan** [comment]
Yeah, it's better to have it configurable, so you can run different apps attached to the same Redis without the overhead of processing the other app messages (even if you use `channel_prefix` you still invoke `#broadcast` in vain).

**Comment by palkan** [comment]
> so you can run different apps attached to the same Redis without the overhead of processing the other app messages

Okay, I was wrong here. This channel is not used for broadcasting (publishing) at all; it seems it's only purpose is to configure the listeners for the real subscriptions. So, there is no need to expose it somehow.
